reconcile/openshift_saas_deploy_trigger_base.py

```diff
@@ -63,13 +63,18 @@ def setup(options):
     return saasherder, jenkins_map, oc_map, settings
 
 
-def construct_tekton_trigger_resource(saas_file_name, env_name, settings,
-                                      integration, integration_version):
+def construct_tekton_trigger_resource(saas_file_name,
+                                      env_name,
+                                      timeout,
+                                      settings,
+                                      integration,
+                                      integration_version):
     """Construct a resource (PipelineRun) to trigger a deployment via Tekton.
 
     Args:
         saas_file_name (string): SaaS file name
         env_name (string): Environment name
+        timeout (Duration): Specifies the timeout before the PipelineRun fails
```

Comment:
Isn't this an integer? It should also specify it is measured in minutes

Reply:
fixed!